client/core/types.go

```diff
@@ -448,6 +455,8 @@ type Exchange struct {
 	Connected     bool                  `json:"connected"`
 	ConfsRequired uint32                `json:"confsrequired"`
 	RegConfirms   *uint32               `json:"confs,omitempty"`
+	Fee           *FeeAsset             `json:"feeAsset"`
+	// RegFees       map[string]*FeeAsset  `json:"regfees"`
```

Comment:
`Fee` could be `FeeQty` at present given the hard-coded assumption of DCR as the fee asset, but I want to float the use of `FeeAsset` so frontend can get use to having it be fully specified like this.